kiwi_lint/empty_class.py

```diff
@@ -0,0 +1,26 @@
+import astroid
+
+from pylint import interfaces
+from pylint import checkers
+from pylint.checkers import utils
+
+
+class EmptyClassChecker(checkers.BaseChecker):
+    __implements__ = (interfaces.IAstroidChecker,)
+
+    name = 'empty-class-checker'
+
+    msgs = {'E4493': ("Remove empty class from git!",
```

Comment:
Use `E4483` code after moving to `empty.py`. This way we keep the ID numbering scheme closer to what already exists.